src/pages/App.tsx

```diff
@@ -65,11 +65,15 @@ const BodyWrapper = styled.div`
   `};
 `
 
-const HeaderWrapper = styled.div`
+const HeaderWrapper = styled.div<{ scrolledState?: boolean }>`
```

Comment:
We only want this scroll behaviour on NFT Pages not all pages

Reply:
switched to be gated by nft flag / phase 1 enabled, as discussed in slack.